components/brave_wallet/browser/simulation_service.cc

```diff
@@ -161,17 +176,17 @@ GURL SimulationService::GetScanTransactionURL(const std::string& chain_id,
   DCHECK(coin == mojom::CoinType::SOL || coin == mojom::CoinType::ETH);
   std::string spec = base::StringPrintf(
       "%s/%s/%s", kBlowfishBaseAPIURL,
-      GetRelativeScanPath(chain_id, coin).c_str(), "transactions");
+      GetRelativeScanPath(chain_id, coin).value_or("").c_str(), "transactions");
```

Comment:
Let's return `GURL()` if there is no such GetRelativeScanPath instead of sending request with double slashes

Reply:
updated: [83c1ea6](https://github.com/brave/brave-core/pull/23385/commits/83c1ea62dc01e8b90300b1cad86f51871e27a654)